sdk/nodejs/provisioning/default.ts

```diff
@@ -0,0 +1,3 @@
+const DEFAULT_IMAGE_REF = 'ghcr.io/dagger/engine:v0.3.4@sha256:666b958a2f716c0d6b22d869c585d6fe07954133ec769bb3d1f310e931cb118f';
```

Comment:
Not a blocker on merge but just FYI we'll want to integrate this with the automatic sdk engine version bumper: https://github.com/dagger/dagger/pull/3809/files